includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -87,35 +87,40 @@ public function __construct(
 
 		// Define setting fields.
 		$this->form_fields = [
-			'enabled'         => [
+			'enabled'                      => [
 				'title'       => __( 'Enable/disable', 'woocommerce-payments' ),
 				'label'       => __( 'Enable WooCommerce Payments', 'woocommerce-payments' ),
 				'type'        => 'checkbox',
 				'description' => '',
 				'default'     => 'no',
 			],
-			'account_details' => [
+			'account_details'              => [
 				'type' => 'account_actions',
 			],
-			'account_status'  => [
+			'account_status'               => [
 				'type' => 'account_status',
 			],
-			'manual_capture'  => [
+			'account_statement_descriptor' => [
+				'type'  => 'account_statement_descriptor',
+				'title' => __( 'Statement Descriptor', 'woocommerce-payments' ),
+				'label' => __( 'Statement descriptor to display in receipt', 'woocommerce-payments' ),
```

Comment:
`label` is not rendered on text fields.
```suggestion
				'type'        => 'account_statement_descriptor',
				'title'       => __( 'Statement Descriptor', 'woocommerce-payments' ),
				'description' => __( 'Statement descriptor to display in receipt.', 'woocommerce-payments' ),
```

Reply:
👍 Fixed in https://github.com/Automattic/woocommerce-payments/pull/788/commits/9341cf604063d5a19c895cc66b3c13cc36740146